src/main/java/org/opensearch/performanceanalyzer/collectors/SearchBackPressureStatsCollector.java

```diff
@@ -266,12 +264,17 @@ public SearchShardTaskStats(
                                 SearchBackPressureStatsValue.Constants
                                         .SEARCHBP_SEARCH_SHARD_TASK_STATS_LIMITREACHEDCOUNT)
                         long limitReachedCount,
+                @JsonProperty(
+                                SearchBackPressureStatsValue.Constants
+                                        .SEARCHBP_SEARCH_SHARD_TASK_STATS_COMPLETIONCOUNT)
+                        long completionCount,
```

Comment:
In case this value is not present in API, this might break the serialization. Consider handling this by desired annotation. Same for below as well.

Reply:
We already set the ser/des to ignore the unknown properties at the `ObjectMapper` level.
```
static {
        mapper = new ObjectMapper();
        mapper.setVisibility(PropertyAccessor.FIELD, JsonAutoDetect.Visibility.ANY);
        mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
    }
```